app/src/main/java/model/objects/elevator/Elevator.java

```diff
@@ -1,23 +1,213 @@
 package model.objects.elevator;
 
+import controller.elevatorSystemController.ElevatorSystemSettings;
+import lombok.Getter;
+import lombok.Setter;
 import model.objects.movingObject.MovingObject;
+import tools.Timer;
 import tools.Vector2D;
 
-import java.awt.*;
+import java.util.*;
 
 /*
  * Elevator is storing all requests under and behind his way, the algorithm finds the closest floor by
  * calculating distance to came from floor A to floor B and all intermediate floors.
  */
 public class Elevator extends MovingObject {
-    private final long TIME_TO_STOP_ON_FLOOR = 0;
-    private final int MAX_HUMAN_CAPACITY = 0;
+    public static final int UNEXISTING_FLOOR = 999;
```

Comment:
Czy to jest górne ograniczenie ilości pięter? Jak tak, to czy ono nie powinno znajdować się w pliku settings, tak jak ilość dostępnych wind?